applications/blink_raw/targets/freertos.mips4k.pic32mx.lccbuffer/nbproject/Makefile-default.mk

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
I have added the nbproject directory for review because it (still) doesn't allow me to debug the project, nor does it clean the target directory upon request (only the application directory). Maybe your IDE or "nbproject" is different? I'm using MPLABX v3.45 for Windows.